src/main/java/com/fvv/bookstore/controller/OrderController.java

```diff
@@ -22,4 +27,53 @@
 	 * @throws ControllerException when a problem in controller happens.
 	 */
 	void addOrder(final Order order) throws OrderValidationException, ControllerException;
+	
+	/**
+	 * Lists all the order by search month.
+	 * 
+	 * @param month to search.
+	 * @param year to search.
+	 * @return a list of orders.
+	 * @throws OrderNotFoundException when not found an order in the database.
+	 * @throws ControllerException when a problem in controller happens.
+	 */
+	List<Order> listTotalOrdersMonth(final Integer month, final Integer year) 
+			throws OrderNotFoundException, ControllerException;
+	
+	/**
+	 * Calculate the total of order amount in the search date.
+	 * 
+	 * @param orders to calculate.
+	 * @return the amount in the search date.
+	 */
+	Double calculateTotalOrders(final List<Order> orders);
+	
+	/**
+	 * Calculate the quantity of orders amount in the search date.
```

Comment:
you are not passing a date for this method. rewrite the javadoc

Reply:
ok